source/includes/steps-fle-configure-the-mongodb-client.yaml

```diff
@@ -80,6 +97,15 @@ content: |
            const patientSchema = {
              "medicalRecords.patients": jsonSchema,
            }
+     .. tab::
+        :tabid: python
+
+        .. code-block:: python
+           :emphasize-lines: 2
+
+           patientSchema = {
+             "medicalRecords.patients": jsonSchema,
```

Comment:
Would it be better to use snake case to stay consistent with other variables like `local_master_key`?

Reply:
I would recommend mentioning it in "Create the MongoDB Client Step 3:
Map the JSON Schema to the Patients Collection" since that is where it is used. Instead of adding any code, you can just mention that they would just assign the value they built in the prior section.

Reply:
Added it to the Create the MongoDB Client Section, under "Map the JSON Schema to the Patients Collection". Here's the updated verbiage:

The MedcoMD engineers assign their schema to a variable. The JSON Schema that MedcoMD defined doesn’t explicitly specify the collection to which it applies. To assign the schema, they map it to the medicalRecords.patients collection namespace: